dbt/include/global_project/macros/adapters/bigquery.sql

```diff
@@ -13,15 +13,14 @@
 
 {% macro cluster_by(raw_cluster_by) %}
   {%- if raw_cluster_by is not none -%}
-  cluster by (
+  cluster by 
   {%- if raw_cluster_by is string -%}
```

Comment:
The test failure is a jinja templating issue. I think if you change this line to `{% if raw_cluster_by is string -%}` (note the missing hyphen) it will fix it. You can remove the extra space after you do that too, I think.